### PR TITLE
fix(tests): Update workflows & Fix failing unit tests

### PR DESCRIPTION
**Pull request by zFernand0**
## Proposed changes

The purpose of this PR is to update all of the V1 workflows and make sure that the unit tests pass under node16 and node18 

## Release Notes

Milestone: N/A

Changelog: 
- tech-debt: updated workflows to remove deprecation warnings. Thanks @t1m0thyj 
- tech-debt: fixed failing unit tests

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [x] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [x] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

File: .github/workflows/zowe-explorer-ci.yml [hunk 1]
```diff
@@ -52,13 +52,13 @@ jobs:
...
       - name: Package VSIX
-        if: matrix.os == 'ubuntu-latest' # && matrix.node-version == '16.x'
+        if: matrix.os == 'ubuntu-latest' && matrix.node-version == '18.x'
         run: yarn package
         working-directory: packages/zowe-explorer
 
       - name: Archive VSIX artifact
-        if: matrix.os == 'ubuntu-latest' # && matrix.node-version == '16.x'
-        uses: actions/upload-artifact@v2
+        if: matrix.os == 'ubuntu-latest' && matrix.node-version == '18.x'
```
Comment (t1m0thyj) [inline .github/workflows/zowe-explorer-ci.yml:60]: On the `main` branch we still archive a VSIX artifact for Node 16. Should we update the logic there as well?

Comment (zFernand0) [reply .github/workflows/zowe-explorer-ci.yml:60]: I think we should 😋

Comment (zFernand0) [reply .github/workflows/zowe-explorer-ci.yml:60]: addressed in 9ca264f44f3581eea919903d05b6baf073e8c023

**Review by rudyflores — APPROVED** [review]
LGTM! thanks @zFernand0 !

**Review by traeok — DISMISSED** [review]
LGTM, thanks @zFernand0!

**Review by rudyflores — APPROVED** [review]
Re-approving this PR after the changes added, thanks @zFernand0 !

**Review by t1m0thyj — APPROVED** [review]

**Pull request merged**